### PR TITLE
Ipb 1063/update pp email address

### PR DESCRIPTION
**Pull request by awhodgson256**
## What does this pull request do?

Adds ability to change email address to the amend a referral flow

## What is the intent behind these changes?

Allow PP user to amend their email address used within R&M

### CODE REVIEW

File: server/models/referralProbationPractitionerEmail.ts [hunk 1]
```diff
@@ -0,0 +1,3 @@
+export interface AmendProbationPractitionerEmailUpdate {
+  ppEmail: string
```
Comment (rgforsyth) [inline server/models/referralProbationPractitionerEmail.ts:2]: I noticed in the service, the dto included:
val preventEmailNotification: Boolean? = null

Should this be included here, or not in the service?
I may have missed something though.

Comment (awhodgson256) [reply server/models/referralProbationPractitionerEmail.ts:2]: It was required by the interface, but I don't have any guidance either way if it's needed or not.  I've assumed we include it but default to null in this case and add an actual value if it becomes necessary in a later change.

@rgforsyth

Comment (rgforsyth) [reply server/models/referralProbationPractitionerEmail.ts:2]: 👍

**Review by rgforsyth — APPROVED** [review]
LGTM

**Pull request merged**